Source/IntegrationTests/TestFiles/LitTests/LitTest/dafny0/Fuel.legacy.dfy.expect

```diff
@@ -21,19 +21,19 @@ Fuel.legacy.dfy(324,21): Related location
 Fuel.legacy.dfy(313,41): Related location
 Fuel.legacy.dfy(335,26): Error: function precondition could not be proved
 Fuel.legacy.dfy(324,21): Related location
-Fuel.legacy.dfy(314,72): Related location
```

Comment:
Someday I want to refactor this test so that the output doesn't get reordered every time we breathe on the verifier. :)